src/library/librarycontrol.cpp

```diff
@@ -302,6 +302,22 @@ LibraryControl::LibraryControl(Library* pLibrary)
                 }
             });
 
+    /// Show the track context menu for selected tracks, or hide it
```

Comment:
I am not sure what doxygen does with comments inside the code. Maybe we should user here just //

Reply:
I had `/// Deprecated controls` in the view so I thought I'd adjust to that.
So in Mixxx cpp files we use /// only for  documentation of inline functions and namespace content?